package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-remote-injection",
-  "version": "0.5.2",
+  "version": "0.6.2",
```

Comment:
Updating the version of the library should rather be part of the release process so I'd kindly ask you to reset this change.